<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MZA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">s</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="openFile">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filePath">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="font">
         <font>
          <family>Sawasdee</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Varianza</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Suma de cuadrados</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Maximo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Minimo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="varLabel">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="covarLabel">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Covarianza</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="maxlabel">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Media</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="mediaLabel">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="minLabel">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="6">
         <widget class="QLabel" name="squareSum">
          <property name="font">
           <font>
            <family>Sawasdee</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
